index.html

```diff
@@ -236,6 +236,32 @@ <h2>
             YES
           </td>
         </tr>
+        <tr data-cite="push-api">
+          <td class="string-token">
+            [="push"=]
+          </td>
+          <td class="is-policy-controlled">
+            YES/NO
+          </td>
+          <td class="is-powerful-feature">
+            YES
+          </td>
+          <td class="spec">
+            [[[push-api]]]
+          </td>
+          <td class="imp-chromium">
+            YES
+          </td>
+          <td class="imp-gecko">
+            YES
+          </td>
+          <td class="imp-webkit">
+            <!-- TODO: update to YES when this ships in 2023, according to:
+              https://www.apple.com/ios/ios-16/features/
+              https://www.apple.com/ipados/ipados-16-preview/features/ -->
+            NO
```

Comment:
It ships on desktop Safari 🎉, so... 

```suggestion
            YES
```

Reply:
oh nice, didn't realize!